simulate/sam.go

```diff
@@ -53,7 +55,7 @@ func IlluminaPairedSam(refName string, ref []dna.Base, numPairs, readLen, avgFra
 
 // generateSamReadNoFlag generates a sam record for the input position.
```

Comment:
Can you update this documentation to reflect that generateSamReadNoFlag now generates not just a sam record but also a []int that is deaminationDistributionSlice?